lib/api/v3/work_packages/schema/work_package_schema_representer.rb

```diff
@@ -0,0 +1,186 @@
+          property_schema :spent_time, type: 'Duration', writable: false
+          property_schema :percentage_done,
+                          type: 'Integer',
+                          title: I18n.t('activerecord.attributes.work_package.done_ratio'),
+                          writable: false
+          property_schema :created_at, type: 'DateTime', writable: false
+          property_schema :updated_at, type: 'DateTime', writable: false
+
+          # non-writable links
+          property_schema :author, type: 'User', writable: false
+          property_schema :project, type: 'Project', writable: false
+          property_schema :type, type: 'Type', writable: false
+
+          # TODO: obey represented.defines_assignable_values? and remove '|| []' from below         @hound: please remember me of that one
+
+          property :assignee,
+                   exec_context: :decorator,
+                   getter: -> (*) {
+                     link = api_v3_paths.available_assignees(represented.project.id)
+                     representer = ::API::Decorators::AllowedValuesByLinkRepresenter.new(
+                       type: 'User',
+                       name: I18n.t('activerecord.attributes.work_package.assigned_to'))
+                     representer.required = false
+
+                     if represented.defines_assignable_values?
+                       representer.allowed_values_href = link
+                     end
+
+                     representer
+                   }
+
+          property :responsible,
+                   exec_context: :decorator,
+                   getter: -> (*) {
+                     link = api_v3_paths.available_responsibles(represented.project.id)
+                     representer = ::API::Decorators::AllowedValuesByLinkRepresenter.new(
+                       type: 'User',
+                       name: I18n.t('activerecord.attributes.work_package.responsible'))
+                     representer.required = false
+
+                     if represented.defines_assignable_values?
+                       representer.allowed_values_href = link
+                     end
+
+                     representer
+                   }
+
+          property :status,
+                   exec_context: :decorator,
+                   getter: -> (*) {
+                     assignable_statuses = represented.assignable_statuses_for(current_user)
+                     representer = ::API::Decorators::AllowedValuesByCollectionRepresenter.new(
+                       type: 'Status',
+                       name: I18n.t('activerecord.attributes.work_package.status'),
+                       current_user: current_user,
+                       value_representer: API::V3::Statuses::StatusRepresenter,
+                       link_factory: -> (status) do
+                           { href: api_v3_paths.status(status.id), title: status.name }
+                         end)
```

Comment:
`end` at 136, 25 is not aligned with `-> (status) do` at 134, 37 or `link_factory: -> (status) do` at 134, 23